### PR TITLE
Ready to pr

### PR DESCRIPTION
**Pull request by MimiXav**

### CODE REVIEW

File: project.xml [hunk 1]
```diff
@@ -188,12 +188,85 @@
...
+            release = "master"
+            test = "fty_common_logging_selftest" >
+
+            <use project = "log4cplus" header = "log4cplus/logger.h" test = "appender_test"
+                repository = "https://github.com/42ity/log4cplus.git"
+                release = "1.1.2-FTY-master" />
+        </use>
+
+        </use>
+    </use>
```
Comment (boricj) [inline project.xml:268]: Duplicated zproject dependencies.

**Review by boricj — APPROVED** [review]
Looks good to me.

File: CMakeLists.txt [hunk 2]
```diff
@@ -236,23 +236,6 @@ ELSE (TNTNET_FOUND)
     message( FATAL_ERROR "tntnet not found." )
 ENDIF (TNTNET_FOUND)
```
Comment (perrettecl) [inline CMakeLists.txt]: Remove CMakeLists.txt from the repo

**Review by perrettecl — CHANGES_REQUESTED** [review]

**Comment by MimiXav** [comment]
Thanks a lot Clement

**Pull request merged**